### PR TITLE
Add a discard example when pattern matching on a list for the tracks-…

### PR DESCRIPTION
**Pull request by aage**
…on-tracks-on-tracks exercise #1015

### CODE REVIEW

File: exercises/concept/tracks-on-tracks-on-tracks/.docs/introduction.md [hunk 1]
```diff
@@ -32,3 +32,19 @@ describe []        // => "Empty list"
...
 describe [5; 7; 9] // => "Non-empty with head: 5"
 ```
+
+You can also _discard_ a value when pattern matching; when you do _not_ care about a value in a specific case (i.e. you aren't going to _use_ a value) you can use an underscore (`'_'`) to signify this:
+
+```fsharp
+let describe list =
+    match list with
+    | [] -> "Empty list"
+    | [x] -> "List with one item"
```
Comment (ErikSchierboom) [inline exercises/concept/tracks-on-tracks-on-tracks/.docs/introduction.md:42]: Maybe add an example where one value is ignored?

```suggestion
    | [x] -> "List with one item"
    | [_; y] -> "List with two items (first item ignored)"
```

**Review by ErikSchierboom — COMMENTED** [review]
One tiny nit.

**Comment by aage** [comment]
@ErikSchierboom I've updated the PR, could you have a look?

**Review by ErikSchierboom — APPROVED** [review]
Nice!

**Pull request merged**